lib/glimesh_web/live/streams_live/following.html.leex

```diff
@@ -2,14 +2,14 @@
     <div class="position-relative overflow-hidden p-3 p-md-5 m-md-3 text-center">
         <div class="col-md-12 mx-auto ">
             <h1 class="display-4 font-weight-normal">
-                <%= gettext("Live Followed Streams") %>
+                <%= gettext("Live Streams") %>
             </h1>
             <%= if length(@channels) == 0 do %>
             <p><%= gettext("None of the streams you follow are live.") %></p>
             <% end %>
         </div>
     </div>
-    <div class="row">
+    <div class="row d-flex justify-content-center">
```

Comment:
I've added "d-flex" and "justify-content-center" so the stream cards will now align with the center of the page if there are less then three on a row.  The previous left-justification felt a bit ugly to me.